app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,8 @@
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
     <%= javascript_importmap_tags %>
-  </head>
+  </head>    
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
```

Comment:
- - Requirement screenshot:
![Selection_049](https://github.com/stino-x/Blog-Rails-/assets/107632780/b3e4ae7b-c1d2-4ac1-8fac-e64062c4fa45)

- [ ] According to the requirement, you are meant to modify your view to look like the given wireframe. Currently, that is not the case in your view files, I suggest you add the correct styling to achieve this. :+1: :smile: 

- - Wireframe:

![Screenshot from 2023-11-29 22-54-21](https://github.com/stino-x/Blog-Rails-/assets/107632780/f998395f-aaee-4513-bfa1-301878c2cacb)

- [ ] Also looking at the screenshot below, you can see that when I try to create a new account, I get the error `Name can't be blank` and this is because the name field is a required param. You need to update the signup form to include a field for name and also accept that in the required params in the controller. :-1: 

- - Scrrenshot reference;
![Selection_047](https://github.com/stino-x/Blog-Rails-/assets/107632780/5d6a4de4-26be-423a-ac3f-c1a289d51bdf)